### DISCUSSION TITLE
How to use silent?

### DISCUSSION THREAD

**Discussion by gagb** [guidance-ai/guidance#548] [Q&A] [answered]
How do I get the code to stop printing to notebook display/clearing the display?
Here's a snippet:

```python
gpt = models.OpenAI("gpt-4", api_key=api_key)
with system():
    lm = gpt + system_message
```

**Comment by Harsha-Nori** [comment, marked as answer]
Apologies, we definitely need to update the documentation here. The best way to suppress all output is to set `echo=False` in the init of the model:

```python
gpt = models.OpenAI("gpt-4", api_key=api_key, echo=False)
with system():
    lm = gpt + system_message
```

**Reply by gagb** [reply]
Thanks Harsha, that worked :)
